### ISSUE TITLE
Omit display of "No information about prerecorded audio is available"

### ISSUE THREAD

**Issue by gautierchomel** [edrlab/thorium-reader#3563]
https://w3c.github.io/cg-reports/publishingcg/CG-FINAL-epub-techniques-20250422/#instructions-1 let implementer choose:

> ELSE either omit this display field if metadata is missing or display "No information about prerecorded audio is available"[ID: ways-of-reading-prerecorded-audio-no-metadata].

We should omit this information as it is strange to have it display on so many ebooks that are just textuals.
